usr/lib/common/loadsave.c

```diff
@@ -1759,6 +1759,7 @@ static CK_RV aes_256_wrap(unsigned char out[40],
 {
     CK_RV rc;
     int outlen;
+    unsigned char buffer[40 + 16]; // add 1 block for openssl internal usage
```

Comment:
use AES block size constant/define (instead of 16). That would make the code more easy to understand, and you might not need the comment.

Reply:
Changed code according to OpenSSL manpage.